crates/ide/src/hover/render.rs

```diff
@@ -239,22 +239,20 @@ pub(super) fn keyword(
     }
     let parent = token.parent()?;
     let famous_defs = FamousDefs(sema, sema.scope(&parent).krate());
-    let keyword_mod = if token.kind() == T![fn] && ast::FnPtrType::cast(parent).is_some() {
-        // treat fn keyword inside function pointer type as primitive
-        format!("prim_{}", token.text())
-    } else {
-        // std exposes {}_keyword modules with docstrings on the root to document keywords
-        format!("{}_keyword", token.text())
-    };
-    let doc_owner = find_std_module(&famous_defs, &keyword_mod)?;
+
+    // some keywords get fancy type tooltips if they are apart of an expression, which require some extra work
+    // panic safety: we just checked that token is a keyword, and we have it's parent in scope, so it must have a parent
```

Comment:
Instead of the comment we should just pass the `parent` to the function as well, then we don't have to do the expect dance in the function later